### DISCUSSION TITLE
Building a Virtual Music Studio

### DISCUSSION THREAD

**Discussion by Mlaughbaum** [jamulussoftware/jamulus#2114] [Ideas]
I’m writing here at the suggestion of a colleague. An in-person music educator since 1990, I’ve been successfully teaching voice and piano lessons online since Covid hit. Most of my students and their parents prefer the lack of driving and I’ve decided to close my brick and mortar space at the end of November and will soon be opening MobileMusicWorkshop.com - a virtual music studio. 

I’ve been using Zoom these past two years but recently found muzie.live (basically Zoom for music education only). It’s a great environment, almost exactly what I’d envisioned in my minds eye, but I’m finding it isn’t consistent and they’ve made the decision to not touch the latency issues. I also jam almost nightly on Jamkazam.com (similar to Jamulus) where they’re really cracking the latency nut but don’t have a user friendly interface or whiteboard function I need to properly educate my students.

As neither one of these sites satisfies my needs entirely, my dream is to marry/meld their two concepts: think Zoom meets Jamulus. I recently met a famous opera coach and performer who thinks my idea could revolutionize his industry as well. On line low(er) latency auditions, lessons, etc.

So in a nutshell: I’m definitely at the starting gate, looking for a road map and fellow “travelers” to help create a music education/jamming/auditions portal using the best of Jamkazam/Jamulus plus Muzie.live and a few other Zoom enhancements I’d rather not live without. Anyone else finding themselves in need of the same thing?

**Comment by pljones** [comment]
Try watching WorldJam - that's Zoom meets Jamulus.  Although that's "Zoom and Jamulus used in a live broadcast" set up.  But people happily use Jitsi and Jamulus.

**Comment by chrisrimple** [comment]
What @pljones said.  My piano teacher has successfully been using Zoom (no audio) + Jamulus throughout the pandemic.

**Reply by Mlaughbaum** [reply]
Thanks for the reply! Can you please elaborate? For each lesson do you have to open both apps/sites or have they created their own site where the apps already work in conjunction and it’s super easy for students join? If separate, is it difficult to navigate both apps at the same time? Some of my students are younger and some have very little patience for tech setup/issues.

**Reply by gene96817** [reply]
My voice teacher uses Zoom. However, the sound is only in one direction. He plays the piano (I listen) then I sing (and he listens). If I need an accompaniment, I play the recording and all the sound goes from my microphone to him. In this case, no Jamulus is needed.

**Reply by Mlaughbaum** [reply]
Thanks for the response. I currently use a mix of Zoom and Muzie to teach voice and piano and my workflow it exactly as you describe: A back and forth sharing. It works ok but what I’m suggesting I’d like to add to my virtual lesson space is the ability to play in near real time with my students (as one would on Jamulus or Jamkazam) so we can practice harmonies, clap rhythms, and play duets together in real time. I’d teach on one of those low latency audio platforms but none have all the cool features such as the whiteboard that I can access on Zoom or Muzie.

**Comment by gene96817** [comment]
I want to follow your progress closely. I am especially interested in more music education resources across the rural/urban divide. As an extension to the rural/urban challenge, we have multiple islands here. One island holds the capital city and the other islands have an extreme case of the rural challenge. (They can't get to the capital by driving.) I don't know if I have the skills to help.

**Reply by Mlaughbaum** [reply]
Thanks for your interest! Where are you located? 😊

**Reply by gene96817** [reply]
I am in Honolulu, Hawaii.

I am interested in Jamulus+Jitsi instead of Jamulus+Zoom for two reasons. Some scenarios would like to avoid paying for a Zoom subscription. AND, the nearest Zoom server is 3000+ miles away. All our Zoom calls have to cross the Pacific and back. With Jitsi (open source), I can run the server locally.

I really want to show some music teachers Jamulus+Jitsi because it would allow the teacher to play music with their students. For me, my teacher could simply play the measures he wants me to sing. Can't do that with Zoom!!!
